New Checkout Pro documentation (#1301)
mercadopago/devsite-docs: guides/online-payments/checkout-pro/advanced-integration.en.md

# Advanced Integration

> Checkout Pro has extra features that allow you to optimize its integration and improve the management of your sales payments.

Reviewer: ```suggestion
Checkout Pro has features that allow you to improve the management of your sales payments through an optimized integration.
```

## Receive payment notifications

> IPN notifications (_Instant Payment Notification_) are automatic notifications about creating new payments and status updates, letting you know if transactions have been approved, declined, or are pending.

Reviewer: ```suggestion
IPN notifications (_Instant Payment Notification_) are automatic notifications that can be sent when there are payments and/or status updates, letting you know if transactions have been approved, declined, or are pending.
```

[Receive IPN notifications](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/notifications/ipn)

Automatic notifications allow you to manage your inventory and keep your system in sync with your business payment flows. Learn how to receive IPN notifications [here](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/notifications/ipn)

## Enter additional information to the preference

> Improve the payment approval and shopping experience in Checkout Pro by adding information to your preferences that details the item purchased and the purchasing user.

Reviewer: ```suggestion
Improve the payment approval and shopping experience in Checkout Pro by adding information that details the item purchased and the purchasing user to your preferences.
```

## Prevent payment rejection

A payment can be rejected because the payment method issuer has detected a problem with the flow, such as a lack of necessary security requirements.

Avoid rejected payments with our [recommendations](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/account/payment-rejections) and improve the approval process.

## Manage cancellations and returns

----[mla, mlm, mco, mlu, mlb, mlc]----

> Cancellations are made when payment has not been completed before the due date and the seller then decides to cancel it. Returns, in turn, happen when the buyer completes the payment, but the seller decides to cancel it totally or partially.

Reviewer: ```suggestion
Cancellations happen when a payment has not been completed before the due date and the seller then decides to cancel it. Returns, in turn, happen when the buyer completes the payment, but the seller decides to cancel it totally or partially.
```

------------

----[mpe]----

> Cancellations are made when payment has not been completed before the due date, and the seller then decides to cancel it. Returns, in turn, happen when the buyer completes the payment, but the seller decides to cancel it totally.

Reviewer: ```suggestion
Cancellations happen when payment has not been completed before the due date, and the seller then decides to cancel it. Returns, in turn, happen when the buyer completes the payment, but the seller decides to cancel it totally.
```

------------

For more information, visit our documentation [on how to manage your payment cancellations and returns](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/ account/cancellations-and-refunds).

## Manage chargebacks

A chargeback occurs when the buyer contacts the entity that the card issuer and communicates that they do not recognize the payment. In practice, this means that the money from this payment will be withheld from your Mercado Pago account until the situation is resolved.

Access our [documentation](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/guides/manage-account/account/chargebacks) and learn how to manage payment chargebacks.

---

### Next step

> > LEFT_BUTTON_REQUIRED_EN

Reviewer 2: Ahhhh, sabemos que es opcional pero si no lo ponemos como requiired el usuario nunca va a llegar a "test integration" por ejemplo no? Está bien entonces :D
